frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -39,6 +50,9 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
                 linkTitle={title}
                 popupTitle={popupTitle}
                 className="co-status-card__popup"
+                position={isHCPresent ? PopoverPosition.bottom : PopoverPosition.top}
+                maxWidth={isHCPresent ? '40rem' : '21rem'}
```

Comment:
Same here use maxWidth as prop
```suggestion
                maxWidth={maxWidth ??  21rem}
```